packages/neuron-ui/src/components/SUDTMigrateToNewAccountDialog/index.tsx

```diff
@@ -84,31 +86,40 @@ const SUDTMigrateToNewAccountDialog = ({
         })
       }
     })
-  }, [cell, t, closeDialog, walletID, tokenInfo, dispatch, sudtAmount])
+  }, [cell, t, onCancel, walletID, tokenInfo, dispatch, sudtAmount])
+
+  const renderList = fields.map(field => (
```

Comment:
How about add `balance` to fields, and when `field.key` is `balance`, return 
```typescript
<TextField label={t(`migrate-sudt.balance`)} field="balance" value={sudtAmount.amount} required disabled />
```
IMO, use `renderList.splice` is hard to understand.